File: CodeWars/sort_numbers.js
function solution(nums){
    if (nums === null ) {
      return [];
    } else {
      return nums.sort((a,b)=>a-b);
    }
  }

  console.log(solution([1, 2, 10, 50, 5]));
  console.log(solution(null));

// Better Solution#1 
function solution(nums){
    return (nums || []).sort(function(a, b){
      return a - b
    });
  }

File: Edabit/sort_numbers.java
import java.util.Arrays;

public class Challenge {
    public static void main(String[] args){
        System.out.println(Arrays.toString(sortNumsAscending(new int[]{1, 2, 10, 50, 5})));
    }

    public static int[] sortNumsAscending(int[] nums) {
        Arrays.sort(nums);
        return nums;
    }

}

// Other Solution#1
import java.util.Arrays;

public class Program {
    public static int[] sortNumsAscending(int[] nums) {
        return Arrays.stream(nums).sorted().toArray();
    }
}